exec/stats.c

```diff
@@ -500,6 +632,11 @@ static void stats_map_notify_fn(uint32_t event, char *key, void *old_value, void
 		return ;
 	}
 
+	/* Ignore schedmiss trackers as the values are read from the circular buffer */
+	if (strncmp(tracker->key_name, "stats.schedmiss", 15) == 0 ) {
```

Comment:
Eventho it is quite readable I would rather like to have a `strlen` here - "stats.schedmiss" is used multiple times in source code so maybe candidate for `#define`